strawberry/schema/config.py

```diff
@@ -1,6 +1,17 @@
+from __future__ import annotations
+
 from dataclasses import dataclass
 
+from .name_converter import NameConverter
+
 
 @dataclass
 class StrawberryConfig:
-    auto_camel_case: bool = True
+    name_converter: NameConverter
+
+    def __init__(
+        self,
+        auto_camel_case: bool = True,
+        name_converter: NameConverter = None,
```

Comment:
we might deprecate `auto_camel_case` in future (for this class)

we should add a docstring here explaining that auto_camel_case is ignored if passing auto_camel_case

Reply:
or use post init